src/Components/ServiceScene/Breakdowns/FieldsBreakdownScene.tsx

```diff
@@ -340,28 +355,23 @@ function getStyles(theme: GrafanaTheme2) {
   };
 }
 
-const avgFields = ['duration', 'count', 'total', 'bytes'];
+const avgFieldTypes = ['duration'];
```

Comment:
I've no context, why did you remove the other fields? Do we need the array?

Reply:
Good shout. On thing that changed is that we now make a decision based on the "type" the API returns, not based on the field name anymore. But I was in fact missing "bytes" here.